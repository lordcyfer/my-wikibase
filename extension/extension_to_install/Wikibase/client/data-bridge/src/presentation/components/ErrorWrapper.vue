<template>
	<section class="wb-db-error">
		<h1>An error occurred</h1>
	</section>
</template>

<script lang="ts">
import Vue from 'vue';
import Component from 'vue-class-component';

@Component
export default class ErrorWrapper extends Vue {}
</script>

<style lang="scss">
.wb-db-error {
	@include dialogBody();
}
</style>
