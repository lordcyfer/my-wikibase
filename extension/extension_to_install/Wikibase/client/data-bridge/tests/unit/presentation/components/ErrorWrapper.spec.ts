import ErrorWrapper from '@/presentation/components/ErrorWrapper.vue';
import { shallowMount } from '@vue/test-utils';

describe( 'ErrorWrapper', () => {
	it( 'renders correctly', () => {
		const wrapper = shallowMount( ErrorWrapper );
		expect( wrapper.element ).toMatchSnapshot();
	} );
} );
