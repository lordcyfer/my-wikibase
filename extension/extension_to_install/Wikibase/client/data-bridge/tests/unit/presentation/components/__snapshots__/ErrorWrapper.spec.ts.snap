// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorWrapper renders correctly 1`] = `
<section
  class="wb-db-error"
>
  <h1>
    An error occurred
  </h1>
</section>
`;
